core-services/prow/02_config/_plugins.yaml

```diff
@@ -570,6 +570,11 @@ cherry_pick_unapproved:
 config_updater:
   gzip: false
   maps:
+    ci-operator/config/**/*-7*.yaml:
```

Comment:
That's covering way too much, please limit the pattern up to the appropriate org or even better, a repo